src/types/rule.ts

```diff
@@ -0,0 +1,4 @@
+interface Rule{
+    key: string;
+    value: string;
+}
```

Comment:
This interface is good for searching the constitution hash table, but not for reading the constitution csv, so we probably should create another interface for that.

Reply:
it's a part of @aviv0102's task